docs/sdks/client-sdks/ios/initialization.mdx

```diff
@@ -0,0 +1,205 @@
+    let assignmentDictionary: [String: Any] = [
+        "allocation": assignment.allocation,
+        "experiment": assignment.experiment,
+        "featureFlag": assignment.featureFlag,
+        "variation": assignment.variation,
+        "subject": assignment.subject,
+        "timestamp": assignment.timestamp
+    ]
+
+    analytics.track(
+        name: "Eppo Assignment", 
+        properties: TrackProperties(assignmentDictionary)
+    )
+}
+```
+
+### Initialization Modes
+
+The SDK supports different initialization modes to suit your needs:
+
+#### Standard Initialization
+The default initialization mode fetches configurations from Eppo's CDN:
+
+```swift
+Task {
+    try await EppoClient.initialize(sdkKey: "SDK-KEY-FROM-DASHBOARD");
+}
+```
+
+#### Initialization with Pre-fetched Configuration
+You can pass in a pre-fetched flag configuration JSON string:
+
+```swift
+Task {
+    try await EppoClient.initialize(
+        sdkKey: "SDK-KEY-FROM-DASHBOARD",
+        initialConfiguration: try Configuration(
+            flagsConfigurationJson: Data(#"{ "pre-loaded-JSON": "passed in here" }"#.utf8),
+            obfuscated: false
+        )
+    );
+}
+```
+
+This will still perform a network request to fetch the latest flag configurations. The provided configuration will be used until the network request is successful.
+
+#### Offline Initialization
+If you'd like to initialize Eppo's client without performing a network request, you can use the `initializeOffline` method:
+
+```swift
+try EppoClient.initializeOffline(
+    sdkKey: "SDK-KEY-FROM-DASHBOARD",
+    initialConfiguration: try Configuration(
+        flagsConfigurationJson: Data(#"{ "pre-loaded-JSON": "passed in here" }"#.utf8),
+        obfuscated: false
+    )
+);
+```
+
+The `obfuscated` parameter is used to inform the SDK if the flag configuration is obfuscated.
+
+### Configuration Updates
+
+#### Fetching Updated Configurations
+After the client has been initialized, you can use `load()` to asynchronously fetch the latest flag configuration from the remote source:
+
+```swift
+try EppoClient.initializeOffline(
+    sdkKey: "SDK-KEY-FROM-DASHBOARD",
+    initialConfiguration: try Configuration(
+        flagsConfigurationJson: Data(#"{ "pre-loaded-JSON": "passed in here" }"#.utf8),
+        obfuscated: false
+    )
+);
+
+// Later, fetch updated configurations
+Task {
+    try await EppoClient.shared().load();
+}
+```
+
+:::note
+As modern iOS devices have substantial memory, applications are often kept in memory across sessions. This means that the flag configurations are not automatically reloaded on subsequent launches.
+
+It is recommended to use the `load()` method to fetch the latest flag configurations when the application is launched.
+:::
+
+### Assignment Logger Schema
+
+The SDK will invoke the `logAssignment` function with an `Assignment` object that contains the following fields:
+
+<ApiOptionRef 
+  name="timestamp"
+  type="string"
+  defaultValue="undefined"
```

Comment:
empty string for all https://github.com/Eppo-exp/eppo-ios-sdk/blob/main/Sources/eppo/Assignment.swift#L1